packages/i18nify-go/readme.md

```diff
@@ -1,3 +1,30 @@
-# i18nify-go
+## i18nify-go
+_A comprehensive solution written in Go to facilitate internationalization._
 
-_A one-stop solution built in golang to provide internationalization support._
\ No newline at end of file
+## Go Software Development Kit (SDK)
+
+This package is compatible with modern versions of Go in module mode. If you have Go installed, you can add the package to your current development module and its dependencies by running:
+
+`go get github.com/razorpay/i18nify/packages/i18nify-go`
+
+will resolve and add the package to the current development module, along with its dependencies.
+
+Alternatively, you can achieve the same result by importing the package in your code:
+
+`import "github.com/razorpay/i18nify/packages/i18nify-go"`
+
+Then, run go get without any parameters.
+
+To use the latest version of this repository's main branch, execute the following command:
+
+`go get github.com/razorpay/i18nify/packages/i18nify-go@master`
+
+
```

Comment:
Add how country can be accessible, adding minimal examples will help developer how it can be consumed.